src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -34,6 +34,10 @@ security:
       - openid
       - email
       - profile
+  - oidc:
+      - openid
+      - email
+      - profile
 paths:
```

Comment:
Just curious if you want to document the actual `/oauth/..` endpoints in Swagger?